mask_polygon.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "mask_polygon",
+    "summary": "Apply a polygon mask",
+    "description": "Applies a polygon mask to a raster data cube. All pixels for which the point at the pixel center **does not** intersect with any polygon (as defined in the Simple Features standard by the OGC) are replaced. This behaviour can be inverted by setting the parameter `inside` to `true`.\n\nThe pixel values are replaced with the value specified for `replacement`, which defaults to `null` (no data). No data values in `data` will be left untouched by the masking operation.",
```

Comment:
> No data values in `data` will be left untouched by the masking operation.

In my opinion,  no-data values in `data` should still be masked/maskable. It won't make a big difference in practice as the replacement value is null typically. Also, to me, the sentence itself reads as "No values will be untouched ...", which is bit strange. I would just remove the sentence.

Reply:
Removed the sentence.